App.Library/Utility/UpdateChecker.cs

```diff
@@ -0,0 +1,107 @@
+﻿using EduRoam.Connect.Install;
+using Semver;
+
+using System.Net.Http;
+using System.Net;
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Security.Policy;
+using Newtonsoft.Json;
+using App.Library.Models;
+using System.Windows.Media;
+using System.IO;
+using System.Diagnostics;
+using System.Threading;
+
+namespace App.Library.Utility;
+
+public static class UpdateChecker
+{
+    private const string UpdateUrlBase = "{0}/windows/{1}/update.json"; // {0} has to be replaced with the arch
```

Comment:
bedoel je niet {1} has to be replaced?
{0} is de govroam danwel eduroam URL, right?

Reply:
Scherp, komt omdat ik {0} verplaatst heb voor de govroam URL. zal het nog aanpassen